src/embed.ts

```diff
@@ -104,46 +117,62 @@ function newlinesToHardlines(str: string): Doc {
     );
 }
 
-function getEmbedBody(
-    node: Node,
-    parser: 'typescript' | 'css',
-    textToDoc: (text: string, options: object) => Doc,
-    options: ParserOptions,
-) {
+function getSnippedContent(node: Node) {
     const encodedContent = getAttributeTextValue(snippedTagContentAttribute, node);
-    let content = '';
 
     if (encodedContent) {
-        content = Buffer.from(encodedContent, 'base64').toString('utf-8');
+        return Buffer.from(encodedContent, 'base64').toString('utf-8');
+    } else {
+        return '';
     }
+}
 
-    if (isNodeSupportedLanguage(node)) {
-        const indentContent = options.svelteIndentScriptAndStyle;
-
-        try {
-            const indentIfDesired = (doc: Doc) => (indentContent ? indent(doc) : doc);
+function formatBodyContent(
+    content: string,
+    parser: 'typescript' | 'css',
+    textToDoc: (text: string, options: object) => Doc,
+    options: ParserOptions,
+) {
+    const indentContent = options.svelteIndentScriptAndStyle;
+
+    try {
+        const indentIfDesired = (doc: Doc) => (indentContent ? indent(doc) : doc);
```

Comment:
I would prefer to just inline the `options.svelteIndentScriptAndStyle` here, or at least move the `indentContent` into the `try` block

Reply:
yeah, makes sense. i was refactoring that thing back and forth quite a bit, i think it's the remains of an earlier refactoring.